lib/solid_queue/scheduler.rb

```diff
@@ -24,6 +22,8 @@ def metadata
     end
 
     private
+      attr_reader :recurring_schedule
```

Comment:
I'd keep this one public, ok to switch to `attr_reader`, but it doesn't need to be private.